api/go.sum

```diff
@@ -1,43 +1,14 @@
-cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
why we delete so many dependency libraries? 

they are only used for testing?

Reply:
yes. I remove some dependency lib and It's auto generated by `Go Module`. and e2e run as an independent project.